sources/updater/waves/ohno.toml

```
@@ -0,0 +1,10 @@
+# The following represents an "emergency" set of update waves for a rapid
+# deployment. The deployment lasts for 3 hours, with a small initial wave,
+# and then all nodes will be updated after the first hour.
+[[wave]]
+start_after = '1 hour'
+fleet_percentage = 5
+
```

```suggestion
[[wave]]
start_after = '2 hours'
fleet_percentage = 25

```
Feels like we could be smoother between the 1 hour and 3 hours.